wgpu-hal/src/gles/adapter.rs

```diff
@@ -293,6 +323,24 @@ impl super::Adapter {
             ver >= (3, 1),
         );
 
+        let r = renderer.to_lowercase();
+        // Check for Mesa sRGB clear bug. See
+        // [`super::PrivateCapabilities::WORKAROUND_MESA_SRGB_SHADER_CLEAR`].
```

Comment:
This link is wrong, and it's concerning that this wasn't caught.

Reply:
Ah, yeah, it's not actually a doc comment, so it's not checked by the doc tools or anything. I just used the same style. I'll fix it.